examples/miscellaneous/plot_set_output.py

```diff
@@ -114,3 +115,20 @@
 # This resets `transform_output` to its default value to avoid impacting other
 # examples when generating the scikit-learn documentation
 set_config(transform_output="default")
+
+# %%
+# Use `config_context` context manager to adjust the transform output within the context
+# manager. Note that you should use the context manager around the `transform` or
+# `fit_transform` usage, Transformer constructed within the context manager will not
+# be affected.
+import numpy as np
+from sklearn import config_context
+
+scaler = StandardScaler()
+scaler.fit(X_train[num_cols])
+
+with config_context(transform_output="pandas"):
+    X_test_scaled_pd = scaler.transform(X_test[num_cols])
+    assert isinstance(X_test_scaled_pd, pd.DataFrame)
+
+assert isinstance(scaler.transform(X_test[num_cols]), np.ndarray)
```

Comment:
I like `assert` statements for computers but for humans (in an example like this) I find them hard to read. Could we replace them by a comment like so:
```
    # X_test_scaled will contain a dataframe
    X_test_scaled = scaler. transform(X_test[num_cols])

# But now the output will be a numpy array
X_test_scaled = scaler. transform(X_test[num_cols])
```

Reply:
Sounds good - updated.